lib/elixir/lib/agent.ex

```diff
@@ -58,7 +58,7 @@ defmodule Agent do
 
       # Compute in the agent/client
       def get_something(agent) do
-        Agent.get(agent, &(&1)) |> do_something_expensive()
```

Comment:
I believe previous code is correct: we pipe `Agent.get/2` result to `do_something_expensive/1`.